force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -120,7 +172,12 @@ public with sharing class VariablesDataTypesOperators {
 	 * @return The complete sentence as a single String.
 	 */
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
+		if (noun != null && verb != null && endingPunctuation != null) {
+            return 'The ' + noun + ' is ' + verb + endingPunctuation;
+		}
+        else {
+            return null;
+        }
 	}
```

Comment:
Review your indentations and formatting

Reply:
hmm... I wonder if some code didn't translate to GitHub properly. When i refreshed my local copy of main through Git I'm showing : 
![image](https://github.com/Cloud-Code-Academy/module1-variables-data-types-and-operators-rileymeye1/assets/134163111/22b4609a-8c70-4df7-b183-5d7a3d13eb73)

Reply:
Okay this looks good.